api/CustomElementRegistry.json

```diff
@@ -0,0 +1,65 @@
+{
+  "api": {
+    "CustomElementRegistry": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CustomElementRegistry",
+        "description": "The CustomElementRegistry interface provides methods for registering custom elements and querying registered elements.",
+        "support": {
+          "chrome": {
+            "version_added": "33",
+            "notes": "Chrome 36+/Opera 20+ implemented a previous version of Custom Elements (v0) that used `.registerElement()`"
+          },
+          "firefox": {
+            "version_added": true,
+            "notes": "Enabled through the `dom.webcomponents.enabled` preference in `about:config`"
+          },
+          "safari": {
+            "version_added": "10.1",
+            "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'"
+          },
+          "edge": {
+            "version_added": false,
+            "notes": "Under consideration"
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "41",
+            "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'"
+          },
+
+
+          "chrome_android": {
```

Comment:
Please put entries in this order:
webview_android, chrome, chrome_android, edge, edge_mobile, firefox, firefox_android, ie, ie_mobile, opera, opera_android, safari, safari_ios. 

Thank you.

Reply:
Hmmm out of curiosity why webview first @teoli2003? Also firefox_android has no specs as far as caniuse goes. For future reference do we totally remove or put null?

Fixed in: https://github.com/mdn/browser-compat-data/pull/623/commits/f17a21a6ba607dc4786874ecae3dd32fac4aa143